docker-compose.el9.yml

```
@@ -151,9 +151,9 @@ x-rpmbuild:
         protobuf_c_min_version: *protobuf_c_min_version
     proj:
       image: rpmbuild-proj
-      version: &proj_version 9.1.1-2
+      version: &proj_version 9.2.0-2
```

Reset the release back to 1:
```suggestion
      version: &proj_version 9.2.0-1
```